packages/headless/src/test/mock-trigger-execute.ts

```diff
@@ -0,0 +1,11 @@
+import {TriggerExecute} from '../../src/api/search/trigger';
+
+export function buildMockExecuteTrigger(
+  config: Partial<TriggerExecute> = {}
+): TriggerExecute {
+  return {
+    type: 'execute',
+    content: {name: '', params: {} as Array<object>},
```

Comment:
Why is the cast needed? Should params be an empty array instead of an empty object?